electricity/src/elektrodistribucija_parser.rs

```diff
@@ -1,8 +1,21 @@
+use std::sync::OnceLock;
```

Comment:
I would create a function that extracts the date component from the header.

```rust
fn extract_date(header: &str) -> anyhow::Result<String> {
    header
        .split(':')
        .last()
        .map(str::trim)
        .map(ToOwned::to_owned)
        .context("failed to extract date from the header")
}
```

And then I would reuse it in both `get_page_date` function and `lib.rs`.
And in the `lib.rs` I would turn `format_date(get_page_date())?` into `get_page_date(html).and_then(format_date)?`.